source/scss/modules/_person.scss

```diff
@@ -3,12 +3,15 @@
   border: none;
   display: block;
   margin: auto;
-  @include rem( width, 12.5 )
+  @include rem( width, 12.5 );
+  @include sb-media( 45em ) {
+    @include rem( margin-top, 5 );
```

Comment:
Wondering if these wouldn't look better aligned with the top of the header text, with a 1.75rem margin-top instead.

![screenshot 2015-06-11 15 20 36](https://cloud.githubusercontent.com/assets/1403188/8116025/8f51b98a-104d-11e5-9222-e1a0ccf4c5de.png)